pkg/grpc/grpc.go

```diff
@@ -100,17 +100,19 @@ func (g *Manager) StartEndpointsDetection() {
 				}
 			}
 
-			for ep, _ := range endpoints {
+			for ep := range endpoints {
 				conn, teardown, err := g.getGRPCConnection(context.TODO(), ep.String(), "", "", true, false, g.sslEnabled)
 				teardown()
 				state := conn.GetState()
 				if err != nil {
 					klog.Error(err)
 				} else if state == connectivity.Ready || state == connectivity.Idle {
 					g.balancer.Add(ep)
+				} else {
+					g.balancer.Remove(ep)
 				}
 			}
-			time.Sleep(200 * time.Millisecond)
+			time.Sleep(time.Second)
```

Comment:
Why changing to a longer interval?

Reply:
Want to reduce the pressure of requesting dns

Reply:
Is there any good effects after changes like this?

Reply:
Apart from the reasons mentioned above, no